modules/DINO CPU/_posts/2000-02-01-assignment1.md

```diff
@@ -335,7 +332,7 @@ make sure you have a correct implementation first, then you start from modifying
 
 We have implemented some tests for your ALU control unit.
 The general ALU control unit tests are in `src/test/scala/components/ALUControlUnitTest.scala`.
-However, these tests require you to implement the required control for not only the R-type instructions but also for I-Types and loads/stores.
+However, these tests require you to implement the required control for not only the R-type instructions but also for I-Types and loads/stores(it will be in assignment2).
```

Comment:
```suggestion
However, these tests require you to implement the required control for not only the R-type instructions but also for I-Types and loads/stores (this will be in [assignment 2]({{'modules/gem5/assignment2' | relative_url}})).
```
There should always be a space before `(`